oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/NodeDocument.java

```diff
@@ -2181,6 +2171,34 @@ private String getCommitValue(Revision revision) {
         return value;
     }
 
+    /**
+     * Check if there may be more recent values in a previous document and return
+     * null if the latter is the case
+     *
+     * @param localValue value as resolved from local value map
+     * @param local      local value map
+     * @param context    the revision context
+     * @return localValue if it is most recent, null otherwise
+     */
+    private Value checkIsMostRecentCommitted(@Nullable Value localValue,
```

Comment:
Just an opinion but, the way this function works and how it is used above, it may be more intuitive and clean if it just returns a boolean for example.

Reply:
+1, renamed as suggested. wdyt?

Reply:
+1